src/main/java/nl/novi/TechItEasy/Repositories/RemoteControllerRepository.java

```diff
@@ -0,0 +1,11 @@
+package nl.novi.TechItEasy.Repositories;
+
+import nl.novi.TechItEasy.Models.RemoteController;
+import nl.novi.TechItEasy.dtos.IdInputDto;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface RemoteControllerRepository extends JpaRepository<RemoteController, Long> {
+    Optional<RemoteController> findById(IdInputDto remoteControllerId);
```

Comment:
Ik denk niet dat dit werkt, want findById gaat zoeken op het type van het id-veld en dat is Long, terwijl je hier IdInputDto meegeeft. en findById(Long) is al een standaard methode